pkgs/top-level/all-packages.nix

```diff
@@ -11861,6 +11861,8 @@ with pkgs;
 
   par2cmdline = callPackage ../tools/networking/par2cmdline { };
 
+  par2cmdline-turbo = callPackage ../by-name/pa/par2cmdline-turbo/package.nix { };
+
```

Comment:
```suggestion
```
This isn't needed with the package being in `pkgs/by-name`

Reply:
:heavy_check_mark: thanks